test/Repository.js

```diff
@@ -137,6 +139,17 @@ describe("Repository", () => {
       repo.lernaJson.packages = customPackages;
       expect(repo.packageConfigs).toBe(customPackages);
     });
+
+    it("returns workspace packageConfigs", () => {
+      const repo = new Repository(testDirWithWorkspaces);
+      expect(repo.packageConfigs).toEqual(["packages/*"]);
+    });
+
+    it("throws with friendly error if workspaces are not configured", () => {
+      const repo = new Repository(testDir);
+      repo.lernaJson.useWorkspaces = true;
+      expect(() => repo.packageConfigs).toThrowErrorMatchingSnapshot();
```

Comment:
I think we can avoid a solitary snapshot with a concise `.toThrow(/workspaces need to be defined/)` assertion here.

Reply:
And if you use the `"EWORKSPACES"` prefix, the regex can be even simpler: `.toThrow(/EWORKSPACES/)`